plugin-tester-java/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-bin.zip
```

Comment:
should we update https://github.com/akka/akka-grpc/pull/1054/files#diff-ee344aab74dcfb1a70eb10f7d1f113bbL23 as well?

Reply:
Hmm the links goes to this same page .

Reply:
I meant AkkaGrpcPlugin.groovy line ~24

Reply:
I don't think so, there's no need to force users to update their Gradle. As soon there's no breaking changes  - we can easily support 5.6, while using 6.4.1 for building plugin itself. `func.GradleCompatibilitySpec` will ensure that 5.6 is still supported.

Reply:
OK, great!